lib/engine/corporation.rb

```diff
@@ -140,6 +140,10 @@ def share_holders
       @share_holders ||= Hash.new(0)
     end
 
+    def corporate_player_share_holders
```

Comment:
remove this method and pass a flag to line 147 player_share_holders(corporations: true) or whatever

Reply:
Done